roles/confluent.kafka_broker/defaults/main.yml

```diff
@@ -1,7 +1,7 @@
 ---
 # More kafka_broker variables are defined in the confluent.variables role
 
-### Boolean to enable cp-ansible's Custom Log4j Configuration
+### Boolean to reconfigure Kafka's logging with RollingFileAppender and log cleanup
```

Comment:
### Boolean to reconfigure thew Kafka Broker's logging with RollingFileAppender and log cleanup funcationality.